.github/workflows/postman-collection-runner.yml

```diff
@@ -39,12 +46,13 @@ jobs:
         env:
           ADMIN_API_KEY: ${{ secrets.INTEG_ADMIN_API_KEY }}
           BASE_URL: ${{ secrets.INTEG_BASE_URL }}
-          MERCHANT_ID: ${{ secrets.STRIPE_GATEWAY_MERCHANT_ID }}
           CONNECTOR_CONFIG_PATH: ${{ env.CONNECTOR_CONFIG_PATH }}
-
-          # Stripe specific env variables, will be replaced by connector_auth.toml
           GATEWAY_MERCHANT_ID: ${{ secrets.STRIPE_GATEWAY_MERCHANT_ID }}
           GPAY_CERTIFICATE: ${{ secrets.STRIPE_GPAY_CERTIFICATE }}
           GPAY_CERTIFICATE_KEYS: ${{ secrets.STRIPE_GPAY_CERTIFICATE_KEYS }}
-        shell: bash
-        run: ./scripts/postman_test_automation.sh ${{ matrix.connector }}
+        uses: nick-fields/retry@v2
+        with:
+          timeout_minutes: 6
```

Comment:
How is the timeout decided?

Reply:
I had updated that in the description today afternoon. Pasting the same here:
> Retries have been added since collections sometimes fail for no reason and this believed to be inconsistencies with matrix itself. Retrying usually fixes unless the error is specific to the collection. It takes 1 - 2 minutes for a collection to run on average. Since, they all run in parallel, timeout of 3 minutes max would be enough. However, maximum number of retries for any collection to run is set to3. So, timeout of 6 minutes have been set as of now. It can be scaled depending on the necessity.

Reply:
Alright. Resolving this conversation.